### ISSUE TITLE
На странице с соревнованиями сделать ссылку на текущее соревнование.

### ISSUE THREAD

**Issue by rozhkovdmitrii** [rozhkovdmitrii/ski73#23]
повесить обработчик по этой ссылке

**Comment by san40** [comment]
а еще в новом протоколе соревнований (биатлонисты) юношей и девушек перепутали :(

**Comment by rozhkovdmitrii** [comment]
Да, Саш, я видел ... просто пока не дошли руки. Хочешь научу :)?

**Comment by san40** [comment]
Еще нужно добавить дату соревнования. Строка вида "гг.мм.дд. - название соревнований". Это облегчит поиск в дальнейшем.
